src/cljs/harja/views/urakka/laadunseuranta/bonukset.cljs

```diff
@@ -0,0 +1,242 @@
+(ns harja.views.urakka.laadunseuranta.bonukset
+  "Bonuksien käsittely ja luonti"
+  (:require [reagent.core :as r]
+            [tuck.core :as tuck]
+
+            [harja.pvm :as pvm]
+            
+            [harja.domain.laadunseuranta.sanktio :as sanktio-domain]
+            
+            [harja.tiedot.navigaatio :as nav]
+            [harja.tiedot.urakka :as tiedot-urakka]
+            [harja.tiedot.istunto :as istunto]
+            [harja.tiedot.urakka.laadunseuranta.bonukset :as tiedot]
+
+            [harja.ui.yleiset :as yleiset]
+            [harja.ui.lomake :as lomake]
+            [harja.ui.napit :as napit]
+            [harja.ui.liitteet :as liitteet]
+            [harja.ui.varmista-kayttajalta :as varmista-kayttajalta]
+            [harja.ui.debug :as debug]))
+
+(defn- kannasta->lomake
```

Comment:
Koko hommassa tehdään paljon tämänlaista konvertointia edestakaisin, miksi näin pitää tehdä?

Reply:
Ilmeisesti sen takia, että :tallenna-erilliskustannus haluaa datan eri muodossa?

Reply:
jeps, niin haluaa